src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/CreateMergedManifestBuildModelTests.cs

```diff
@@ -32,23 +29,81 @@ class CreateMergedManifestBuildModelTests
         private static readonly string buildNumber = "azDevBuildNumber";
         private static readonly string sourceBranch = "thisIsASourceBranch";
         private static readonly string commitSourceVersion = "thisIsASourceVersion";
+        private static readonly string id = "1234";
+        private static readonly string version = "thisIsVersion";
+        private static readonly string noVersion = "thisIsNoVersion";
 
-        private readonly AssetData assetDataWithoutName = new AssetData(true)
-        { Version = "noNameVersion" };
+        private PackageArtifactModel package1 = new PackageArtifactModel
+        {
+            Attributes = new Dictionary<string, string>
+                {
+                    { "NonShipping", "true" }
+                },
+            Id = id,
+            Version = version
+        };
+
+        private PackageArtifactModel package2 = new PackageArtifactModel
+        {
+            Attributes = new Dictionary<string, string>
+                {
+                    { "NonShipping", "true" }
+                },
+            Id = id ,
+            Version = noVersion
+        };
+
+        private PackageArtifactModel packageWithNoVersion = new PackageArtifactModel
+        {
+            Attributes = new Dictionary<string, string>
+                {
+                    { "NonShipping", "true" }
+                },
+            Id = id,
+            Version = null
+        };
+
+        private PackageArtifactModel packageNonShipping = new PackageArtifactModel
+        {
+            Attributes = new Dictionary<string, string>
+                {
+                    { "NonShipping", "false" }
+                },
+            Id = id,
+            Version = null
+        }; 
 
-        private readonly AssetData assetDataWithoutVersion = new AssetData(true)
-        { Name = "noVersionData" };
+        List<PackageArtifactModel> packages;
+        List<BlobArtifactModel> blobs;
+
+        private BlobArtifactModel blob1 = new BlobArtifactModel
```

Comment:
It looks like there's a distinction between blob1, blob2  and blob3 in the categories and nonshipping properties. They should probably be renamed to make that distinction clear if it's something that matters for the tests. That should be done throughout the class.

Reply:
Renamed all the blobs and packages in this file